opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/test/resources/application.yml

```diff
@@ -0,0 +1,91 @@
+server:
```

Comment:
This file is not necessary, you should modify 
opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/resources/application-test-one-time-postgres-ramfs.yml
OR
opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/resources/application-test-mocked-sandbox.yml
if necessary

Reply:
File removed, opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/resources/application-test-one-time-postgres-ramfs.yml was modified